docs/README.rst

```diff
@@ -6,5 +6,6 @@ these commands suffice::
   cd dask/docs
   conda create -n daskdocs --file requirements-docs.txt
   source activate daskdocs
+  Update the conf.py file with the code path (e.g. sys.path.insert(0, os.path.abspath('../../')) )
```

Comment:
Could you try doing an editable install from the root of the `dask` repo?

    python -m pip install -e .

That should avoid having to modify the conf file.